### ISSUE TITLE
something wrong with the test of SE-ResNet-50?

### ISSUE THREAD

**Issue by cai-y13** [hujie-frank/SENet#5]
Thanks for sharing such excellent work. 

But when I test the SE-ResNet-50 caffemodel, I have encountered some problems.

I add the data layer at the bottom and the accuracy layer at the top of the prototxt:
layer {  
  name: "data"  
  type: "Data"  
  top: "data"  
  top: "label"  
  include {  
    phase: TEST  
  }  
  transform_param {  
    mirror: false  
    crop_size: 224  
    mean_value: 103.939  
    mean_value: 116.779  
    mean_value: 123.68  
  }  
  data_param {  
    source: "examples/imagenet/ilsvrc12_val_lmdb"  
    batch_size: 10  
    backend: LMDB  
  }  
}

layer {
  name: "accuracy/top1"
  type: "Accuracy"
  bottom: "classifier"
  bottom: "label"
  top: "accuracy/top1"
  include {
    phase: TEST
  }
}

layer {
  name: "accuracy/top5"
  type: "Accuracy"
  bottom: "classifier"
  bottom: "label"
  top: "accuracy/top5"
  include {
    phase: TEST
  }
  accuracy_param {  
    top_k: 5  
  }
}

and run the test program on ILSVRC12 imagenet dataset:
./build/tools/caffe test --model=models/SE-resnet-50/SE-ResNet-50.prototxt --weights=models/SE-resnet-50/SE-ResNet-50.caffemodel --iterations 5000 -gpu=0 
(test batch_size=10)

but i get the following result:
I0830 16:03:59.629042 37849 caffe.cpp:313] Batch 0, accuracy/top1 = 0
I0830 16:03:59.629132 37849 caffe.cpp:313] Batch 0, accuracy/top5 = 0
I0830 16:03:59.629142 37849 caffe.cpp:313] Batch 0, loss = 8.86858
almost all of the accuracy are 0.

Very grateful and hope that getting some advice on this issue from you. Thanks very much.

**Comment by hujie-frank** [comment]
The validation images have not been changed since ILSVRC12, but the order of labels have been changed since ILSVRC13. You can take the order-changed validation list [here](https://github.com/hujie-frank/SENet/blob/master/ILSVRC2017_val.txt). In addition,  the mean values we used are [104, 117, 123]  rather than decimals, which we listed at the beginning of each model prototxt.

And if you want to achieve exact results, I suggest you using ImageDataLayer, resizing the short edge of input images to 256 (To convenient implementation ,  you can do a small modification to function ReadImageToCVMat() in [src/caffe/util/io.cpp](https://github.com/BVLC/caffe/blob/master/src/caffe/util/io.cpp) ) and than coducting center crop 224x224  as described  in the caption of Table 1.

Looking forward your testing feedback.

**Comment by cai-y13** [comment]
Thanks so much. Following your suggestions, I replace the labels, and get an top-1 accuracy of 76.96%, and top-5 accuracy of 93.42%. It's really a great improve compared with the baseline, but a little lower than the proposed result. (I have resize the images to 256x256) Is there any other tricks when doing the testing?

Thanks again for your suggestions and such excellent work.

**Comment by tianhaijie** [comment]
@cai-y13 Hi!can u give me a train list since the order of labels has been changed.thanks!
